docs/ABOUT.md

```diff
@@ -0,0 +1,7 @@
+Nim is a general-purpose language that is designed to be efficient, expressive, and elegant. Nim is statically typed and compiled but lets you write elegant code that runs efficiently. Nim code compiles to C, C++, or JavaScript letting you target many platforms. Nim is inspired by Python and shares many of Python's characteristics.
+
+Nim gives you a lot of the convenience you find in Python and other languages like easy to read syntax, a package management system for using new libraries in your projects, and user-friendly trackbacks for debugging errors.
+
+Nim gives you the speed and portability of C, but with a high-performance garbage-collecter and elegant syntax that makes the language very approachable to new people.
+
+Designed and developed by Andreas Rumpf and first appearing in 2008, Nim is still considered to be relatively a new programing language.
```

Comment:
I don't know if new is the right term - maybe niche. A decade isn't exactly new.